### PR TITLE
Migrate indexer docs

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+# NEAR Data Flow
+
+## Video explanation
+
+In the video below we have tried to give a quick overview on main concept of how the data is flowing in NEAR Protocol blockchain. You will find a text version below the video.
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VSBJ-A69Km4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (gagdiez) [inline docs/1.concepts/data-flow/near-data-flow.md:12]: We can remove the title and description, and put the video at full width.

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+## Text version
```
Comment (gagdiez) [inline docs/1.concepts/data-flow/near-data-flow.md:15]: We can remove the title and the explanation that the text and video are the same.

File: docs/1.concepts/data-flow/token-transfer-flow.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 2
+sidebar_label: Token Transfer
+---
+
+# Token Transfer flow
+
+:::warning Pre-requisites
```
Comment (gagdiez) [inline docs/1.concepts/data-flow/token-transfer-flow.md:8]: Since they are all part of the same sub-menu, we do not need this warning

File: website/sidebars.json [hunk 4]
```diff
@@ -35,6 +35,13 @@
         }
       ]
     },
+    {
+      "Data Flow": [
+        "concepts/data-flow/near-data-flow",
+        "concepts/data-flow/token-balance",
+        "concepts/data-flow/token-transfer-flow"
```
Comment (gagdiez) [inline website/sidebars.json:42]: from the text, it looks like token-transfer-flow goes before token-balance

**Review by gagdiez — CHANGES_REQUESTED** [review]

**Comment by gagdiez** [comment]
tldr: 
- We had already a structure in place for the indexers in "Develop", we should use that one to not change the structure of docs.
- There is one page in concepts that is too technical and we can remove.

File: docs/1.concepts/data-flow/token-transfer-flow.md (hunk 3 above)
Comment (bucanero) [reply docs/1.concepts/data-flow/token-transfer-flow.md:8]: removed.

File: docs/1.concepts/data-flow/near-data-flow.md (hunk 2 above)
Comment (bucanero) [reply docs/1.concepts/data-flow/near-data-flow.md:15]: removed

File: docs/1.concepts/data-flow/near-data-flow.md (hunk 1 above)
Comment (bucanero) [reply docs/1.concepts/data-flow/near-data-flow.md:12]: fixed

File: website/sidebars.json (hunk 4 above)
Comment (bucanero) [reply website/sidebars.json:42]: fixed

File: docs/1.concepts/data-flow/token-balance.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
+---
```
[the remark this answers is not in the document]
Comment (bucanero) [reply docs/1.concepts/data-flow/token-balance.md:1]: we can check with the indexer team if they want to keep it for now, or we can remove it.

File: website/sidebars.json [hunk 6]
```diff
@@ -201,6 +208,21 @@
...
     },
+    {
+      "type": "html",
+      "value": "<span class='menu__link'><b><small> Indexing Tools </small></b></span>"
+    },
+    "tools/indexer/near-indexer-for-explorer",
+    "tools/indexer/near-lake-indexer",
+    {
+      "NEAR Lake": [
+        "tutorials/indexer/migrating-to-near-lake-framework"
```
[the remark this answers is not in the document]
Comment (bucanero) [reply website/sidebars.json]: moved to "Monitor user activity" section

File: docs/4.tools/indexer/near-indexer-for-explorer.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# NEAR Indexer for Explorer
```
[the remark this answers is not in the document]
Comment (bucanero) [reply docs/4.tools/indexer/near-indexer-for-explorer.md:8]: content merged, page removed.

File: docs/4.tools/indexer/near-lake-indexer.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 5
+---
+
+# NEAR Lake Indexer
```
[the remark this answers is not in the document]
Comment (bucanero) [reply docs/4.tools/indexer/near-lake-indexer.md:5]: content merged, page removed.

File: website/sidebars.json [hunk 9]
```diff
@@ -201,6 +208,21 @@
       "type": "html",
       "value": "<hr/>"
     },
+    {
+      "type": "html",
+      "value": "<span class='menu__link'><b><small> Indexing Tools </small></b></span>"
```
[the remark this answers is not in the document]
Comment (bucanero) [reply website/sidebars.json:214]: section removed

**Comment by bucanero** [comment]
from Pavel:
`change title on the left and Header of https://near-docs-pr-1236.onrender.com/concepts/advanced/indexers to "Introduction to Indexers"`
